src/puppetlabs/i18n/core.clj

```diff
@@ -14,6 +14,15 @@
            (.getResources "locales.clj")
            enumeration-seq))
 
+(defn parse-info-file
+  [info-file-path]
+  (let [data-structure (-> info-file-path
+                           slurp
+                           read-string)]
```

Comment:
(clojure.core's read-string will eval clojure code, which we probably don't want)